src/components/AddColleague/add-colleague.component.tsx

```diff
@@ -11,17 +10,19 @@ export const AddColleague: React.FC = (props) => {
     setName(event.target.value)
   }
 
-  function handleCreate() {
+  function handleAddCollleague() {
     dispatch(createColleague({
       name,
       favorite: false
     }))
   }
 
   return (
-    <div className={styles['container']}>
-      <input className={styles['input']} onChange={handleChange} />
-      <button className={styles['button']} onClick={handleCreate}>Create</button>
+    <div className="input-group shadow-sm">
+      <input type="text" className="form-control" onChange={handleChange} placeholder="Name" />
+      <div className="input-group-append">
+        <button type="button" className="btn btn-outline-secondary" onClick={handleAddCollleague}>Add</button>
```

Comment:
a global classokat erdemes atemelni?

Reply:
sztem utility classoknal alapvetoen jo ha global a scope, nem kell mindenre local. Es igy egy ertekelesi szempont is lehet hogy mire hasznal global stilust es mire nem, meg hogy osszekeveredik-e benne :) 

Annak meg nem latom ertelmet hogy bootstrapbol atcopyzzuk azokat a classokat amiket hasznalunk mert elore ugy se tudjuk hogy mit szeretne hasznalni aki epp megoldja.

Reply:
hat ezzel nem teljesen ertek egyet, a style encapsulatonnel inkabb az az irany, hogy ha utility classaid vannak, akkor utility componenseid legyenek es azt inheritald, pl <SecondaryButton />

Reply:
De milyen utility componenst tudsz irni egy `text-align: center`, `display: flex` stilusokra? Nem talalom mar a cikket, de volt egy erdekes iras arrol hogy mennyire csokkkentettek a css meretet azzal hogy ahelyett hogy mindig full izolalt stilusokat irtak helyette utility classokat definialtak egyszeru pozicionalasokra, margora stb. es localban tenyleg csak a specificikus dolgok maradnak.
En nem latom hogy ez hogy lenne megoldhato komponensekkel, ez igy elegge nem scalel
```
 <FlexContainer>
   <AlignCenter>
     <Button />
   </AlignCenter>
 </FlexContainer>
```